tick/stateful/node_evaluator.go

```diff
@@ -34,7 +34,9 @@ type NodeEvaluator interface {
 	EvalRegex(scope *Scope, executionState ExecutionState) (*regexp.Regexp, error)
 	EvalTime(scope *Scope, executionState ExecutionState) (time.Time, error)
 	EvalDuration(scope *Scope, executionState ExecutionState) (time.Duration, error)
+	EvalMissing(scope *Scope, executionState ExecutionState) (*ast.Missing, error)
```

Comment:
Again here, I don't think we need this? Are we calling it anywhere I did't see calls to it anywhere.